content/metadata.yaml

```diff
@@ -434,23 +434,55 @@ authors:
       - Data Curation
     affiliations:
       - Division of Oncology, Children's Hospital of Philadelphia
-  - name: Sharon J. Diskin
+  - github: rebkau
+    name: Rebecca S. Kaufman
+    initials: RSK
+    orcid: 0000-0001-8535-9730
+    email: kaufmanr1@chop.edu
+    contributions:
+      - Formal Analysis
+      - Investigation
+      - Validation
+    affiliations:
+      - Division of Oncology, Children's Hospital of Philadelphia
+  - name: Zalman Vaksman
```

Comment:
There is a Zalman Vaksman in ORCID: https://orcid.org/0000-0001-5121-8657, can you check if we should use this? Also, I assume there is a CHOP email; do we want to use that instead?

Reply:
He is no longer at CHOP, and he is in Israel at the moment - I can send him a message asking about the ORCID, but I think not necessary at this point since I cannot readily identify him there without pubs.